srcgl/glgtk/GLCapability.d

```diff
@@ -85,14 +85,6 @@ template GLCapability()
 		return true;
 	}
 	
-	/**
-	 * The widget should use this method to redraw it self at any time
-	 */
-	bool drawFrame()
```

Comment:
It's my false. I just want to get rid of the warning message while building demos/gl/simple/SimpleGL.d:

E:\GtkD\Sources\srcgl\glgtk\GLCapability.d(80): Deprecation: function gtk.Widget.Widget.addOnDraw is deprecated


The second patch can fix this.